AK/Vector.h

```diff
@@ -86,22 +86,26 @@ requires(!IsRvalueReference<T>) class Vector {
         other.reset_capacity();
     }
 
+#ifdef KERNEL
+    Vector(Vector const&) = delete;
+
+    template<size_t other_inline_capacity>
+    Vector(Vector<T, other_inline_capacity> const&) = delete;
```

Comment:
`delete("Reason")` should be available in the used c++ functions

Reply:
Yeah I wanted to use that but it's only available since C++26 (our standard is currently set to C++23).

We could bump it to C++26, but I don't know if all compilers (especially OSS-Fuzz and Apple Clang) support -std=c++26 yet. Just bumping it for the non-lagom build seems a bit meh tbh.